db/model/room.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/room.js
+ *
+ * Rooms are gathering point for incident response. This table
+ * will hold the basic details for each of these rooms.
+ */
+
+const constants = require('../constants');
+
+const assoc = {};
+
+module.exports = function user(seq, dataTypes) {
+  const Room = seq.define('Room', {
+    id: {
+      type: dataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    name: {
+      type: dataTypes.STRING,
+      allowNull: false,
+      unique: true,
+      validate: {
+        is: constants.nameRegex,
+      },
+      comment: 'Create a named room ',
+    },
+    active: {
+      type: dataTypes.BOOLEAN,
+      defaultValue: false,
+      comment: 'Determines if room is still active',
+    },
```

Comment:
okay, I can see this being present on all the other models. Let's disregard then.